src/api/app/views/webui/shared/_requests_state_and_type_form.html.haml

```diff
@@ -0,0 +1,12 @@
+%p
+  Display
+  = select_tag('request_type_select',
+               options_for_select(@available_types, @default_request_type),
+               data: { table: 'all_requests_table' })
+  requests in state
+  = select_tag('request_state_select',
+               options_for_select(@available_states, @default_request_state),
+               data: { table: 'all_requests_table' })
+  %span#request_count
+  \:
+  = image_tag('ajax-loader.gif', id: 'spinner', class: 'hidden')
```

Comment:
This is [`.invisible`](https://getbootstrap.com/docs/4.1/utilities/visibility/) in bootstrap...